packages/smart-router/evm/getBestTradeWithStableSwap.ts

```diff
@@ -43,9 +43,12 @@ export async function getBestTradeWithStableSwap(
     if (stableSwapPair) {
       // Get latest output amount from v2 and use it as input to get output amount from stable swap
       const stableInputAmount = await getLatestOutputAmount()
+      const midStableInputAmount = CurrencyAmount.fromRawAmount(baseTrade.inputAmount.currency, '1')
```

Comment:
We may need to define how we should estimate mid price first.
Also need to consider the decimal difference between input and output token here